src/main/java/com/ldtteam/structurize/storage/SurvivalBlueprintHandlers.java

```diff
@@ -0,0 +1,68 @@
+package com.ldtteam.structurize.storage;
+
+import com.ldtteam.structurize.blueprints.v1.Blueprint;
+import com.ldtteam.structurize.util.PlacementSettings;
+import net.minecraft.client.multiplayer.ClientLevel;
+import net.minecraft.core.BlockPos;
+import net.minecraft.world.entity.player.Player;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Class to register a survival blueprint handler.
+ */
+public class SurvivalBlueprintHandlers
+{
+    /**
+     * Map of handlers.
+     */
+    private static final Map<String, ISurvivalBlueprintHandler> handlers = new HashMap<>();
+
+    /**
+     * Get a specific handler by id.
+     * @param id the id of the handler.
+     * @return the handler instance.
+     */
+    public static ISurvivalBlueprintHandler getHandler(final String id)
+    {
+        return handlers.get(id);
+    }
+
+    /**
+     * Getthe full list of handlers.
+     * @return a list.
+     */
+    public static List<ISurvivalBlueprintHandler> getHandlers()
+    {
+        return new ArrayList<>(handlers.values());
```

Comment:
Make this an unmodifyable list, store that unmodifyable view as a field. Makes calls to this way cheaper.